lbry/wallet/usage_payment.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+import logging
+
+from lbry.wallet import Wallet
+from lbry.wallet.ledger import Ledger
+from lbry.wallet.dewies import lbc_to_dewies
+from lbry.wallet.transaction import Output, Transaction
+
+log = logging.getLogger(__name__)
+
+
+class WalletServerPayer:
+    PAYMENT_PERIOD = 24 * 60 * 60
+
+    def __init__(self, ledger: Ledger, wallet: Wallet):
+        self.ledger = ledger
+        self.wallet = wallet
+        self.running = False
+        self.task = None
+
+    async def pay(self):
+        while self.running:
+            await asyncio.sleep(self.PAYMENT_PERIOD)
+            features = await self.ledger.network.get_server_features()
+            address = features['payment_address']
+            amount = str(features['daily_fee'])
+            if not address or not amount:
+                continue
+
+            if not self.ledger.is_valid_address(address):
+                raise Exception(f"Invalid address: {address}")
+            if self.wallet.is_locked:
+                raise Exception("Cannot spend funds with locked wallet")
```

Comment:
I think this will kill the pay service, even after the wallet is unlocked no payment will be sent. Perhaps just print a message and continue running?